src/components/grid/gridRowContent.js

```diff
@@ -26,6 +36,11 @@ export const GridRowContent = props => {
   const gridRowContentStyles = theme.get('gridItem.gridRowContent')
   const locationName = useSessionLocation(session)
   const column2Styles = gridRowContentStyles.column2
+  const sessionState = useSessionState(session)
+  let labelToDisplay = sessionState
+  
+  //Don't diplay select and Read_Only labels as per requirement in zen-626
+  if (sessionState == SESSION_BOOKING_STATES.SELECT || sessionState == SESSION_BOOKING_STATES.READ_ONLY) { labelToDisplay = ''}
```

Comment:
you can do the same logic in a simpler and functional way like this:

```suggestion
  const labelToDisplay = (sessionState === SESSION_BOOKING_STATES.SELECT || sessionState === SESSION_BOOKING_STATES.READ_ONLY)
    ? ''
    : sessionState
```

* also I changed boolean expression to use the strict-equality operator `===`. We prefer that over the abstract equality operator, `==`, which is more complex. `==` does type coercion first, then checks for equality. This requires the reader to know the type coercion tables which are long and not very intuitive. It's also inconsistent with every other programming language in the world, so even though `==` works just fine here, I think it's just better we stick with `===` for consistency and clarity.

This mdn doc is useful: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness

Reply:
Done . And duly noted . I have read a lot of comments about this online as well. DA for me for using ==. Thanks